ruby-service/lib/request_validator.rb

```
@@ -0,0 +1,131 @@
+require 'json'
+
+class RequestValidator
+  VALIDATION_ERRORS = []
```

RuboCop (1.88.0)

[convention] Freeze mutable objects assigned to constants.

(Style/MutableConstant)